src/FlightMap/MapItems/PlanMapItems.qml

```diff
@@ -27,9 +27,9 @@ Item {
 
     property var    _map:                       map
     property var    _vehicle:                   vehicle
-    property var    _missionController:         planMasterController.missionController
-    property var    _geoFenceController:        planMasterController.geoFenceController
-    property var    _rallyPointController:      planMasterController.rallyPointController
+    property var    _missionController:         masterController.missionController
+    property var    _geoFenceController:        masterController.geoFenceController
+    property var    _rallyPointController:      masterController.rallyPointController
```

Comment:
I need to stare at this a little more to make sure. But I believe this change is incorrect. This should continue to the planMasterController which is passed in as a property. The bug is in FlyViewMap.qml. The planMasterController property on PlanMapItems in there should be bound to masterController which is the controller associated with each vehicle.

Reply:
Done: https://github.com/mavlink/qgroundcontrol/pull/9757/commits/1bba400819eb7ac92d51044f3efb168b4eacfb38